download_games.py

```diff
@@ -18,15 +20,41 @@
                     help="substantially reduce the number of logged messages")
 parser.add_argument("--max", metavar="MAX", default="60",
                     help="max number of games")
+parser.add_argument("--site", metavar="SITE", default="lichess", help="Website to query user game data")
+
 settings = parser.parse_args()
-logging.basicConfig(format="%(message)s", level=settings.loglevel, stream=sys.stdout)
 
+logging.basicConfig(format="%(message)s", level=settings.loglevel, stream=sys.stdout)
 logging.debug("Downloading games from: " + settings.username)
 
-response = requests.get(
-    'https://lichess.org/api/games/user/' + settings.username + '?max=' + settings.max + '&token=' + settings.token + '&perfType=blitz,rapid,classical&opening=true')
-pgn = str(response.text)
-all_games = open("games.pgn", "w")
-all_games.write(pgn)
-all_games.close()
-logging.debug("Finished. Pgn is in games.pgn ")
+
+if settings.site == "lichess":
+    with open("games.pgn", "w") as new_file:
+        myclient = lichess.Client()
+        new_file.write(myclient.export_by_user(settings.username, max_games=settings.max))
+    new_file.close()
+    logging.debug("Finished. Pgn is in games.pgn ")
+elif settings.site == "chessdotcom":
+    url_chessdotcom = "https://api.chess.com/pub/player/" + \
+                      settings.username.lower() + "/games"
+    url_archive = url_chessdotcom + "/archives"
+    http_response = urllib.request.urlopen(url_archive)
+
+    archives = http_response.read().decode("utf-8")
+    archives = archives.replace("{\"archives\":[\"", "\",\"")
+    archive_dates = archives.split("\",\"" + url_chessdotcom)
+    archive_dates[len(archive_dates) - 1] = archive_dates[
+        len(archive_dates) - 1].rstrip("\"]}")
+
+    with open("games.pgn", "w") as new_file:
```

Comment:
I think we will need the `max` param also for chess.com
One way to do it is use parse the json from `https://api.chess.com/pub/player/hikaru/games/2017/04` to count the number of games. and get the pgn with `.pgn`